README.md

```diff
@@ -101,9 +101,9 @@ Significant [Contributors to OpenPDF](https://github.com/LibrePDF/OpenPDF/graphs
   [@lapo-luchini](https://github.com/lapo-luchini)  
   [@jeffrey-easyesi](https://github.com/jeffrey-easyesi)  
   [@V-F](https://github.com/V-F)     
-  [@sixdouglas](https://github.com/sixdouglas) - Douglas Six
+  [@sixdouglas](https://github.com/sixdouglas) - Douglas Six     
```

Comment:
Spaces?

Reply:
The spaces here are meant to be displayed as new line in the Markdown file. There are some at the end of every other lines for this purpose.